### ISSUE TITLE
NOTE for "target" (definition) has wrong indication of excepting 'included area" WCAG 2.1 and 2.2

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2424]
I'll add definition in 2.2, but also in 2.1 this is wrongly written...
https://w3c.github.io/wcag/guidelines/22/#dfn-targets

The definition goes like:

> target
> region of the display that will accept a pointer action, such as the interactive area of a user interface component
> 
> NOTE
> If two or more touch targets are overlapping, the overlapping area should not be included in the measurement of the target size, except when the overlapping targets perform the same action or open the same page.

Now in the NOTE it says: **"the overlapping area should not be included in the measurement of the target size"**

I think this is not correct and only counts for the target "hidden / obscured" by the other.
The (overlapping) area for the target 'on top' should be fine to include in the size...

**Comment by mbgower** [comment]
@jake-abma Did you add the definition to 2.2?

**Comment by jake-abma** [comment]
Hi @mbgower what about:

> target
> region of the display that will accept a pointer action, such as the interactive area of a user interface component
> 
> NOTE
> If two or more touch targets are overlapping, the overlapped area of the underlying user interface component should not be included in the measurement of the target size, except when the overlapping targets perform the same action or open the same page.

**Comment by patrickhlauke** [comment]
can we drop the word "touch" please? it's not just about touch

**Comment by patrickhlauke** [comment]
ah, as this sounded familiar, i see i already made that tweak https://github.com/w3c/wcag/pull/1485

**Comment by jake-abma** [comment]
Good point @patrickhlauke so change to:

> target
> region of the display that will accept a pointer action, such as the interactive area of a user interface component
> 
> NOTE
> If two or more targets are overlapping, the overlapped area of the underlying target should not be included in the measurement of the target size, except when the overlapping targets perform the same action or open the same page.
